### PR TITLE
Portfolio: setup and mobile first

### PR DESCRIPTION
**Pull request by Nelofarzabi**
Understand how to parse a Figma design to create a UI.
Flexbox to place elements in the page.
Build a personal portfolio site.
Use images and backgrounds to enhance the look of the website.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+      <nav class="nav_item">
+        <ion-icon class="menu" name="menu-sharp"></ion-icon>
+      </nav>
+    </header>
+
+    <section class="hero">
+      <h2 class="name">
+        I'm Nelofar Zabi<br />Glad to see<br />
+        you!
+      </h2>
```
Comment (usorfaitheloho) [inline index.html]: Weldone on implementing some of the project requirements, please the main title of your webpage should be a `h1` property for semantics reasons and search engine optimization 😄

File: index.html [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+  <body>
+    <header class="flex-container">
+      <div class="logo-container">
+        <a class="logo" href="">My Logo</a>
+      </div>
+      <nav class="nav_item">
+        <ion-icon class="menu" name="menu-sharp"></ion-icon>
+      </nav>
+    </header>
```
Comment (usorfaitheloho) [inline index.html:19]: Kindly increase the width of your hamburger menu icon to meet the required design template, as this looks too small and may not meet a user requirement 😄 

Figma design :
image

Your design :
image

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
> Hi @Nelofarzabi  ,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

  [ ](https://camo.githubusercontent.com/679757134a8fee76cfe1941f9f3ecb0e0cf56dd02da4498268ce31f0dacdfca3/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)
> 
> ## Highlights
> * Good design✔️
> * Followed HTML and CSS best practices ✔️
> * Used GitHub flow ✔️
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by wayungi — APPROVED** [review]
Hi @Nelofarzabi , 

image

Great work, your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**Remember: Every finish line 🏁  is the beginning of a new race 🐎** 

## Highlights
- [ ] All linter checks are passing ✔️ 
- [ ] Good commit messages ✔
- [ ]  Correct GitHub flow ✔️ 
- [ ] Descriptive PR body ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**